docs/rfcs/004-issueish-list.md

```diff
@@ -0,0 +1,112 @@
+# Issueish List
+
+## Status
+
+Proposed
+
+## Summary
+
+Display a list of all open pull requests in the current repository in the GitHub tab.
+
+## Motivation
+
+To provide a navigational element that makes sense even if you aren't on an active feature branch.
+
+To give users a way to see an overview of what's going on in the repository.
+
+As an initial building block toward a pull request review workflow.
+
+## Explanation
+
+### Accordion Lists
+
+Within the GitHub panel, render a vertical stack of two collapsible lists of _issueish_ (pull request or issue) items:
+
```

Comment:
reading this, it wasn't immediately clear to me that the number was "time."  I saw "3M" and thought "3 million"?  So can we clarify that label somehow?  Also is this the date that the pull request was opened, or the date of the first commit in that pull request?

Reply:
I thought the same thing, and was going to comment on it before realizing it is using `M` to signify months since `m` would be taken by minutes.

Reply:
yeah, ideally we could use "minutes", "weeks", "etc" instead of abbreviations.  Moment.js has a "humanize time since" function that provides this in an i18n-friendly manner.  https://momentjs.com/docs/#/durations/humanize/